ext/ITensorNetworksEinExprsExt/src/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,58 @@
+module ITensorNetworksEinExprsExt
+
+using ITensors: Index, ITensor, Algorithm
+using ITensorNetworks:
+  ITensorNetworks,
+  ITensorNetwork,
+  vertextype,
+  vertex_data,
+  externalinds,
+  contraction_sequence
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function to_einexpr(tn::ITensorNetwork)
+  IndexType = Any
+  VertexType = vertextype(tn)
+
+  tensor_exprs = EinExpr{IndexType}[]
+  tensor_inds_to_vertex = Dict{Set{IndexType},VertexType}()
+  inds_dims = Dict{IndexType,Int}()
+
+  for v in vertices(tn)
+    tensor_v = tn[v]
+    inds_v = collect(inds(tensor_v))
+    push!(tensor_exprs, EinExpr{IndexType}(; head=inds_v))
+    tensor_inds_to_vertex[Set(inds_v)] = key
+    merge!(inds_dims, Dict(inds_v .=> size(tensor_v)))
+  end
+
+  externalinds_tn = collect(externalinds(tn))
+  expr = SizedEinExpr(sum(tensor_exprs; skip=externalinds_tn), inds_dims)
+
+  return expr, tensor_inds_to_vertex
+end
```

Comment:
I realized that there isn't really a need for this one function to make both `expr` and `tensor_inds_to_vertex`, can we just split it in two? The part building `expr` could be called `expr(tn)` (or whatever seems appropriate for that) and the part building `tensor_inds_to_vertex` could be it's own function called `tensor_inds_to_vertex(tn)`. I think that would help settle the question of what to call it.

Reply:
Done!